From 0df127a931fcfb68d08ce592ddd31b3aa1d48e27 Mon Sep 17 00:00:00 2001
From: Peter O'Connor <peter@solus-project.com>
Date: Tue, 10 Jul 2018 12:13:52 +1000
Subject: [PATCH] Fix python path, as python still in lib directory

Python currently sits it /usr/lib until eopkg is removed. However, everything
else uses lib64 which confuses it.

Signed-off-by: Peter O'Connor <peter@solus-project.com>
---
 scripts/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/CMakeLists.txt b/scripts/CMakeLists.txt
index 4398527..c4eb48a 100644
--- a/scripts/CMakeLists.txt
+++ b/scripts/CMakeLists.txt
@@ -43,8 +43,8 @@ if(NOT LLDB_BUILD_FRAMEWORK)
     set(swig_python_subdir python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
   endif()
 
-  set(SWIG_PYTHON_DIR ${LLVM_LIBRARY_OUTPUT_INTDIR}/${swig_python_subdir})
   set(SWIG_INSTALL_DIR lib${LLVM_LIBDIR_SUFFIX})
+  set(SWIG_PYTHON_DIR ${CMAKE_BINARY_DIR}/lib/python${PYTHON_VERSION_MAJOR}.${PYTHON_VERSION_MINOR})
 
   # Install the LLDB python module
   install(DIRECTORY ${SWIG_PYTHON_DIR} DESTINATION ${SWIG_INSTALL_DIR})
-- 
2.22.0




